crypt.go

```diff
@@ -321,7 +321,7 @@ func unsealKeyFromTPM(tpm *TPMConnection, k *SealedKeyObject, pin string) ([]byt
 		// has a null authorization value, then this will allow us to unseal the key without requiring any type of manual recovery. If the
 		// storage hierarchy has a non-null authorization value, ProvionTPM will fail. If the TPM owner has changed, ProvisionTPM might
 		// succeed, but UnsealFromTPM will fail with InvalidKeyFileError when retried.
-		if pErr := ProvisionTPM(tpm, ProvisionModeWithoutLockout, nil); pErr == nil {
```

Comment:
the comment should maybe be expanded to cover the new || condition

Reply:
I've actually changed this in https://github.com/snapcore/secboot/pull/112 to just ignore the error instead, as there's not really any harm in attempting to unseal again even if provisioning fails with an error.